View/FrequencyRelationships/GraphicalControl.xaml.cs

```diff
@@ -9,5 +9,10 @@ public GraphicalControl()
             InitializeComponent();
             
         }
+
+        private void UserControl_DataContextChanged(object sender, System.Windows.DependencyPropertyChangedEventArgs e)
```

Comment:
What's this guy doing here? Is this necessary to fix the bug with the radio buttons?

Reply:
Good catch, i think we added that to help try to diagnose the radio button issue. I deleted it. A good reason to always put "todo: delete me".